IA shuffle and edit of Contribute section (#13998)
PrefectHQ/prefect: docs/3.0rc/community/developing-a-new-worker-type.mdx

Prefect workers set up execution infrastructure and start flow runs.

<Tip>
**Advanced topic**

> This guide is for users who want to extend the Prefect framework. Completing this successfully requires deep knowledge of 

Reviewer: ```suggestion
Extending the Prefect framework by developing a worker requires deep knowledge of 
```

> Prefect concepts. For standard use cases, we recommend using one of the [available workers](/3.0rc/deploy/dynamic-infra/control-runs/#worker-types) instead.

Reviewer: ```suggestion
Prefect. For standard needs, we recommend using one of the [available worker types](/3.0rc/deploy/dynamic-infra/control-runs/#worker-types).
```
